erpnext/manufacturing/doctype/manufacturing_settings/manufacturing_settings.json

```diff
@@ -34,6 +34,15 @@
    "precision": ""
   }, 
   {
+   "default": "BOM", 
+   "fieldname": "update_fg_based_on", 
+   "fieldtype": "Select", 
+   "label": "Update FG Based On", 
```

Comment:
Backflush Raw Materials Based On (better label?)

- BOM
- Material Transferred for Manufacture

@rmehta please suggest